src/mpid/ch4/netmod/ucx/ucx_recv.h

```diff
@@ -85,7 +85,7 @@ MPL_STATIC_INLINE_PREFIX void MPIDI_UCX_mrecv_cmpl_cb(void *request, ucs_status_
     } else {
         if (unlikely(status == UCS_ERR_MESSAGE_TRUNCATED)) {
             /* FIXME: we have no way of passing the tag bits back in this case */
-            ucp_request->req = (void *) UCS_ERR_MESSAGE_TRUNCATED;
+            ucp_request->status = UCS_ERR_MESSAGE_TRUNCATED;
```

Comment:
Remove the FIXME comment?

Reply:
This change does not affect the status of this FIXME comment.

Reply:
I see.